scalar/uuid.go

```diff
@@ -63,8 +63,15 @@ func (s *UUID) Set(src any) error {
 		return s.Set(value2)
 	case [16]byte:
```

Comment:
I wonder if `uuid.UUID` should also be checked?...

Reply:
It works since `uuid.UUID` implements `Stringer` interface